panoptes_client/workflow.py

```diff
@@ -115,6 +117,22 @@ def remove_subject_sets(self, subject_sets):
 
         return self.links.subject_sets.remove(subject_sets)
 
+    def subject_set_status(self, subject_set_id):
+        """
+        A generator which yields :py:class:`.SubjectWorkflowStatus` objects for subjects in the
+        subject set of the given workflow
+
+        Examples::
+
+            for status in workflow.subject_set_status(1234):
+                print(status.retirement_reason)
+        """
+
+        subjects = SetMemberSubject.where(subject_set_id=subject_set_id).object_list
```

Comment:
Does `object_list` return all the pages of result set data from the API or just this current page of data? 

It seems like iterating over the result set (all pages) and constructing the list of ids is more aligned to the way we use these resources in this library already? 
```
subjects = []
for sms in SetMemberSubject.where(subject_set_id=self.id):
    subjects.append(sms.subject_id)
```

Reply:
ah crud. I believe you are right, `object_list` returns current page of data as opposed to all data. https://github.com/zooniverse/panoptes-python-client/blob/b25c3b88ce29ae0c19597119b456c799f9d41426/panoptes_client/panoptes.py#L884 DOH! 
I'll update with the fix. Thanks for catching that